### PR TITLE
added home page content and profile pictures

### PR DESCRIPTION
**Pull request by ashwoodaa**
it is telling me that I can't automatically merge

### CODE REVIEW

**Comment by ashwoodaa** [comment]
I made sure to pull before I did this to make sure I had everything that was up. I checked my status and it said everything was good. I can't figure out the conflict. Will look into it tomorrow.

File: index.html [hunk 1]
```diff
@@ -26,26 +26,33 @@
...
 
     </div><!-- Column center -->
 
-    <div id="">
-      <div class=""><img src="" alt=""></div>
-      <div class="">
-        <p></p>
+    <div id="columnCenter">
+      <div id="Joe">
+        <div class="profilePics"><img src="joeProfilePic.jpg" alt="Joe"></div></div><!--feel free to change-->
```
Comment (josephkane) [inline index.html:46]: I think the image source url needs to be "img/joeProfilePic.jpg" etc

Comment (ashwoodaa) [inline index.html:46]: cool. taken care of :+1:

**Comment by MeganWord** [comment]
:+1: nice work!

**Pull request closed without merging**